allennlp/data/tokenizers/pretrained_transformer_tokenizer.py

```diff
@@ -22,36 +24,98 @@ class PretrainedTransformerTokenizer(Tokenizer):
     We take a model name as an input parameter, which we will pass to
     ``AutoTokenizer.from_pretrained``.
 
+    We also add special tokens relative to the pretrained model and trunkate the sequences.
+
+    This tokenizer also indexes tokens and adds the indexes as the ``Token`` fields so that
+    they can be picked up by SingleIdTokenizer
+
     Parameters
     ----------
     model_name : ``str``
         The name of the pretrained wordpiece tokenizer to use.
     start_tokens : ``List[str]``, optional
-        If given, these tokens will be added to the beginning of every string we tokenize.  We try
-        to be a little bit smart about defaults here - e.g., if your model name contains ``bert``,
-        we by default add ``[CLS]`` at the beginning and ``[SEP]`` at the end.
+        If given, these tokens will be added to the beginning of every string we tokenize.
```

Comment:
I'd probably just remove these parameters, as `add_special_tokens` covers the intended use for `start_tokens` and `end_tokens`.  If we really want to keep them, we need to note that these get added _before_ adding special tokens.